src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.scss

```diff
@@ -12,6 +12,38 @@ Ace Editor
         &__editor {
             flex: 1 1 auto;
         }
+
+        .inline-feedback-d-none {
+            display: none;
+        }
+
+        .inline-feedback {
+            display: block;
+        }
+
+        .btn-inline-comment {
+            display: none;
+            color: #3e8acc; // btn-primary color
+            height: 16px;
+        }
+
+        .ace_gutter-cell {
+            position: relative;
+        }
+
+        .ace_gutter-cell:hover .btn-inline-comment {
+            display: inline-flex;
+            position: absolute;
+            top: 8px; // standard gutter height is 16px
+            left: 25%;
+            transform: translate(-50%, -50%);
+
+            fa-icon:hover {
+                font-size: 15px;
+                transition: font-size 0.35s ease;
```

Comment:
I think the animation for the icon getting larger could be a bit faster so I would suggest 200ms here, but this is really just personal preference 😄

Reply:
If anyone else is feeling the same, we can change it :)